distributed/tests/test_steal.py

```diff
@@ -982,7 +982,7 @@ async def test_parse_stealing_interval(s, interval, expected):
         ctx = contextlib.nullcontext()
     with ctx:
         ws = WorkStealing(s)
-        s.periodic_callbacks["stealing"].callback_time == expected
+        assert s.periodic_callbacks["stealing"].callback_time == expected
```

Comment:
This test is now failing consistently.

Reply:
Nice catch. I've not looked into this deeply, but this test seems to assume `ws = WorkStealing(s)` will update the stealing periodic callback on the scheduler. Are we sure that's actually happening?

Reply:
I adjusted the test implementation to what I _think_ it would want to test. Let me know what you think!

Reply:
I assume adding the `work-stealing: False` config was the solution here, since it ensured a WorkStealing plugin/extension wasn't already connected to the scheduler?

Reply:
The solution was adding `work-stealing: False` to the config so that there would be no pre-existing `WorkStealing` in place as well as manually starting it.

I _think_ that's the behavior we wanted to test here rather than overwriting some pre-existing `WorkStealing` with the new config value, which would not happen.

Reply:
Yeah I agree that's the behavior this is meant to be testing.